src/popup/modules/QrLib/QrError.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Errors in QR code generation
+ *
+ * @module QrLib/QrErrors
+ */
+
+export class DataOverflowError extends Error {
+    constructor(...rparams) {
+        let params = rparams.length>0
```

Comment:
You do not need to make it so compact. Better be a little more elaborate, but make actual something easily readable.

Better use a real if and only overwrite the first parameter, if given. (See suggestion above.)